lib/rmg/CRmgTemplate.cpp

```diff
@@ -378,24 +378,31 @@ void ZoneOptions::serializeJson(JsonSerializeFormat & handler)
 
 	{
 		//TODO: add support for std::map to serializeEnum
-		static const std::vector<std::string> STRENGTH =
+		static const std::vector<std::string> zoneMonsterStrengths =
 		{
+			"none",
 			"weak",
 			"normal",
 			"strong"
 		};
 
-		si32 rawStrength = 0;
-		if(handler.saving)
-		{
-			rawStrength = static_cast<decltype(rawStrength)>(zoneMonsterStrength);
-			rawStrength++;
-		}
-		handler.serializeEnum("monsters", rawStrength, EMonsterStrength::ZONE_NORMAL + 1, STRENGTH);
-		if(!handler.saving)
+		int temporaryZoneMonsterStrengthIndex = monsterStrength == EMonsterStrength::ZONE_NONE ? 0 : monsterStrength - EMonsterStrength::ZONE_WEAK + 1 ; // temporary until serializeEnum starts supporting std::map
+		// temporaryZoneMonsterStrengthIndex = 0, 1, 2 and 3 for monsterStrength = ZONE_NONE, ZONE_WEAK, ZONE_NORMAL and ZONE_STRONG respectively
+		handler.serializeEnum("monsters", temporaryZoneMonsterStrengthIndex, 2, zoneMonsterStrengths); // default is normal monsters
```

Comment:
am I correctly understand, that before default value was monsters_strong?

Reply:
I just checked on develop and no, the default there is also zone_normal.
I checked that by changing monster strength on players' zones in JC: I gave Blue weak monsters, Teal normal, Green strong and for Red I removed the monster strength field from the template.
Green (strong) got guarded Sawmills.
Teal (normal) got unguarded Sawmills and guarded Campfires.
Blue (weak) got unguarded Sawmills and Campfires.
Red (default) got unguarded Sawmills and guarded Campfires, same as Teal, so default was normal.
The logs did contain the generator's complaint about the missing 'monsters' field, but it did generate the maps.